src/Client.php

```
@@ -10,6 +10,8 @@
 use Psr\Log\LoggerInterface;
 use Psr\Log\NullLogger;
 
+define('TRAVIS_RUNNING', isset($_SERVER['TRAVIS_COMMIT']));
```

Why did you add this? I don't think we can do this, at least you need to check that the constant is not defined before.

---

Sorry for the delay, this fell off of my radar.
I did this because phan was yelling at me https://travis-ci.org/Expensify/Bedrock-PHP/builds/313619228

But you're right, I should check if this is defined first

---

Actually, after taking a 2nd look at the code, I think I should add a comment for phan to ignore the error
```
FAIL! src/Client.php: src/Client.php:436 PhanUndeclaredConstant Reference to undeclared constant \TRAVIS_RUNNING
FAIL! src/Client.php: src/Client.php:640 PhanUndeclaredConstant Reference to undeclared constant \TRAVIS_RUNNING
```